Core/LocalAdmin.cs

```diff
@@ -181,7 +181,17 @@ internal async Task Start(string[] args)
         {
             await LoadJsonOrTerminate();
 
-            if (DataJson!.EulaAccepted == null)
+            var isEulaNotAccepted = DataJson!.EulaAccepted == null;
+
+            var autoEula = Environment.GetEnvironmentVariable("EULA")?.ToUpperInvariant() is "1" or "TRUE"
```

Comment:
If this variable is being set globally, consider changing this to SLLA_EULA.

`SL` - Secret Laboratory
`LA` - LocalAdmin
`EULA` - The EULA no one reads.

Reply:
Done